Dockerfile

```diff
@@ -12,7 +12,7 @@ COPY package.json package-lock.json /app/
 
 # install dependencies
 RUN apk add --no-cache git
-RUN npm ci --only=production
+RUN git config --global url."https://".insteadOf ssh:// && npm ci --only=production
```

Comment:
Also add `ssh` to `https` in Dockerfile so the same process is used while building the container. Otherwise, the build fails because `ssh` is not available in the image.